deployment/docker-compose.yml

```diff
@@ -0,0 +1,124 @@
+    networks:
+     - chatbot-network
+    volumes:
+     - postgres-db-vol:/var/lib/postgresql/data
+     - postgres-db-backup:/var/backups:rw
+    labels:
+      ofelia.enabled: "true"
+      ofelia.job-exec.backup.schedule: "0 0 0 * * *" #"Takes a single dump within the db container every day at 00:00 am"
+      ofelia.job-exec.backup.command: "pg_dump --dbname=postgresql://$POSTGRES_USER:$POSTGRES_PASSWORD@localhost:5432/$POSTGRES_DB  --file=/var/backups/postgres_backup_latest.sql"
+
+  webserver:
+    image: nginx:latest
+    ports:
+     - 80:80
+     - 443:443
+    restart: always
+    expose:
+     - 80
+     - 443
+    depends_on:
+     - chatbot
+    environment:
+      - CHAT_DOMAIN=${DOMAIN:-"localhost"}
+    volumes:
+     - ./nginx/nginx.conf.template:/etc/nginx/templates/default.conf.template:ro
+     # - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+     -  logs-vol:/var/log/nginx/
+     - ./certbot/www:/var/www/certbot/:ro
+     - ./certbot/conf/:/etc/nginx/ssl/:ro
+    networks:
+     - chatbot-network
+
+  certbot:
+    image: certbot/certbot:latest
+    volumes:
+     - ./certbot/www/:/var/www/certbot/:rw
+     - ./certbot/conf/:/etc/letsencrypt/:rw
+    networks:
+     - chatbot-network
+
+  ofelia-scheduler:
+    image: mcuadros/ofelia:v0.3.7
+    depends_on:
+      - postgresvectordb
+    command: daemon --docker
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - postgres-db-backup:/var/backups:r
+      - ./backup:/var/backup-copies/:rw
+    labels:
+      ofelia.job-local.copy-backup-daily.schedule: "0 30 0 * * *" # Copies latest dump from DB contaier every day at 00:30 am
+      ofelia.job-local.copy-backup-daily.command: sh -c "cp /var/backups/postgres_backup_latest.sql /var/backup-copies/postgres_backup_daily_`TZ=\":Asia/Calcutta\" date +%Y-%m-%d_%H_%M_%S`.sql"
+      ofelia.job-local.prune-daily-backup.schedule: "0 45 0 * * *" # Retains only latest 7 of the daily backups 
+      ofelia.job-local.prune-daily-backup.command: sh -c "ls -tr /var/backup-copies/postgres_backup_daily* | head -n -7 |  xargs rm -f --"
+      ofelia.job-local.copy-backup-weekly.schedule: "0 30 0 ? * SUN" # Copies latest dump on every sunday 00:30 am
+      ofelia.job-local.copy-backup-weekly.command: sh -c "cp /var/backups/postgres_backup_latest.sql /var/backup-copies/postgres_backup_weekly_` TZ=\":Asia/Calcutta\" date +%Y-%m-%d_%H_%M_%S`.sql"
+      ofelia.job-local.prune-weekly-backup.schedule: "0 45 0 ? * SUN" # Retains only latest 4 of the weekly backups
+      ofelia.job-local.prune-weekly-backup.command: sh -c "ls -tr /var/backup-copies/postgres_backup_weekly* | head -n -4 |  xargs rm -f --"
+      ofelia.job-local.copy-backup-monthly.schedule: "0 30 0 1 * *" # Copies latest dump on first day of every month
+      ofelia.job-local.copy-backup-monthly.command: sh -c "cp /var/backups/postgres_backup_latest.sql /var/backup-copies/postgres_backup_monthly_` TZ=\":Asia/Calcutta\" date +%Y-%m-%d_%H_%M_%S`.sql"
```

Comment:
` TZ=\` <- should there be a space here before TZ?

Reply:
Down on line 114 as well. 

I ask because line 102 does not have this space.

Reply:
That space wasn't an issue. But removed them anyway.